packages/keyda-demo/src/lib/FormToSubmit/index.js

```diff
@@ -27,16 +27,17 @@ const FormToSubmit = (props) => {
   const handleSubmit = useCallback(
     (e) => {
       e.preventDefault();
-      setTimeout(() => {
-        const formData = new FormData();
-        console.log(formData);
-        formData.append('keyTimeList', keyState.keyTimeList);
-        formData.append('userId', keyState.userId);
-
-        axios.post(REQUEST_URL + suffix, formData).then((response) => {
-          console.log(response);
-        });
+      (async () => {
+        const dataToSubmit = {
+          keyTimeList: keyState.keyTimeList,
+          userId: keyState.userId,
+        };
+        console.log(dataToSubmit);
+        const request = await axios
+          .post(REQUEST_URL + suffix, dataToSubmit)
+          .then((response) => response);
```

Comment:
I changed the logic of post request to our server into IIFE(immediately invoked function expression) to use `async/await` keyword for the proper access to the server.